plugins/fake-bot-after.conf

```diff
@@ -16,13 +16,26 @@
 # Documentation can be found here:
 # https://github.com/coreruleset/fake-bot-plugin
 
-SecRule REQUEST_HEADERS:User-Agent "@pm amazonbot applebot bingbot linkedinbot facebookbot facebookcatalog facebookexternalhit googlebot twitterbot" \
+SecRule TX:FAKE-BOT-PLUGIN_WHITELIST_BROKEN_APPLE_DEVICES "@streq 1" \
     "id:9504110,\
     phase:1,\
+    pass,\
+    t:none,\
+    msg:'Fake Bot Plugin: Broken Apple device detected from %{REMOTE_ADDR} - whitelisting',\
```

Comment:
```suggestion
    msg:'Fake Bot Plugin: Messenger detected from %{REMOTE_ADDR} - whitelisting',\
```

Reply:
I would call it broken as such behavior is... broken.